lib/src/barback/asset_environment.dart

```diff
@@ -179,32 +180,25 @@ class AssetEnvironment {
     var transformers = <Set>[];
 
     if (compiler == Compiler.dartDevc) {
```

Comment:
I think it would be clearer to switch on `compiler` to make it more obvious that dartDevc and dart2Js are mutually exclusive. (Also, I just noticed that the latter should be capitalized `dart2JS`. Two letter acronyms remain all caps.)